transactions.go

```diff
@@ -60,13 +60,13 @@ func (lw *LibWallet) IndexTransactions(beginHeight int32, endHeight int32, after
 		}
 
 		if block.Header != nil {
-			err := lw.txDB.Set(BucketTxInfo, KeyEndBlock, &endHeight)
+			err := lw.txDB.Set(BucketTxInfo, KeyEndBlock, &block.Header.Height)
```

Comment:
The final log message (in line 110 below) will also benefit from this correction.

Reply:
Final log should display current best block height since the wallet scanned through those blocks and did not find related transactions.

Reply:
What if there was an error during the scan, and the wallet did not scan through to the best block?